include/simgrid/host.h

```diff
@@ -26,6 +27,8 @@ XBT_PUBLIC(void*) sg_host_extension_get(sg_host_t host, size_t rank);
 XBT_PUBLIC(sg_host_t) sg_host_by_name(const char *name);
 XBT_PUBLIC(const char*) sg_host_get_name(sg_host_t host);
 XBT_PUBLIC(xbt_dynar_t) sg_hosts_as_dynar();
+XBT_PUBLIC(xbt_swag_t) sg_host_get_process_list(sg_host_t host);
```

Comment:
This is a re-implementation of a function that was there when I started working with SimGrid, but was later removed. The only thing this version does is return host->processes.

If really needed, I can make a function that builds a dynar by iterating through the swag. The major drawback is that I'll need to create and destroy the dynar several times. I can't reuse it because process migrations change the process list.